### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_ViewMessage_Screen reader : NVDA is not reading header "ViewMessage" as heading.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/AzureStorageExplorer#3466]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.3.1;#Win32;#A11ySev2;#Benchmark;#NVDA;#WCAG2.4.6;#JAWS;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10
Screen reader: NVDA Version:2020.2,JAWS(2020.2006.12ILM)

### **Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it..
6. Navigate to View message button and click.
7. Start screen reader as NVDA and check how NVDA reads "View message" .

### **Actual:**
NVDA is not reading header "ViewMessage" as heading.

**This issue is also observed with JAWS**

**Similar issue also observed for following scenarios:**

1. Queues-->AddMessage
2. Queues-->DequeueMessage
3. Queues-->ClearQueue

### **Expected:**
NVDA should read header "ViewMessage" as heading.

### **User Impact:**
User will not understand about the heading present in page.

### **Attachment**
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-prraha_microsoft_com/EcH5lJEE2ENFjphOTZ7FjYABSf5x7Jehq7nYkxXoBR7xuQ?e=NYy6Sw

**Comment by JasonYeMSFT** [comment]
@craxal If the title work you have done have fixed this bug, let's close this one and label it to be verified.

**Comment by craxal** [comment]
@pranshu512 A change has been merged. Please verify if this has been fixed. If not, please provide details about expected results, keystrokes, etc.

**Comment by pranshu512** [comment]
Still,this bug is not fixed.Refer attachment.
here, Expectation is that heading should be assigned to "View Message"


[3466_Reopened.zip](https://github.com/microsoft/AzureStorageExplorer/files/5299344/3466_Reopened.zip)

**Comment by craxal** [comment]
@pranshu512 Please share the build number used to verify. We need to make sure you used the build containing the fix.

**Comment by pranshu512** [comment]
@craxal I tested this issue with NVDA 2020.2 and Storage Explorer 1.16.0 and it not reproducing.Hence,Closing this bug.
Refer attachment


[3466_Closed.zip](https://github.com/microsoft/AzureStorageExplorer/files/5346040/3466_Closed.zip)

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;
